Index: J2EE.iml
===================================================================
--- J2EE.iml	(revision 10d4dc04936a6034a9bcbd6f70cb431408f4ca57)
+++ J2EE.iml	(revision 10d4dc04936a6034a9bcbd6f70cb431408f4ca57)
@@ -1,2 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4" />
\ No newline at end of file
Index: .idea/J2EE.iml
===================================================================
--- .idea/J2EE.iml	(revision 10d4dc04936a6034a9bcbd6f70cb431408f4ca57)
+++ .idea/J2EE.iml	(revision 10d4dc04936a6034a9bcbd6f70cb431408f4ca57)
@@ -1,2 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4" />
\ No newline at end of file
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/bank/log/\r\n/log/\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .gitignore	(revision 10d4dc04936a6034a9bcbd6f70cb431408f4ca57)
+++ .gitignore	(date 1691053721193)
@@ -1,2 +1,12 @@
-/bank/log/
+# IntelliJ project files
+/target/
 /log/
+.idea
+.idea/
+.classpath
+.project
+.settings
+
+*.iml
+out
+gen
\ No newline at end of file
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Default ignored files\r\n/shelf/\r\n/workspace.xml\r\n# Datasource local storage ignored files\r\n/dataSources/\r\n/dataSources.local.xml\r\n# Editor-based HTTP Client requests\r\n/httpRequests/\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/.gitignore	(revision 10d4dc04936a6034a9bcbd6f70cb431408f4ca57)
+++ .idea/.gitignore	(date 1691053729984)
@@ -1,8 +1,12 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
-# Editor-based HTTP Client requests
-/httpRequests/
+# IntelliJ project files
+/target/
+/log/
+.idea
+.idea/
+.classpath
+.project
+.settings
+
+*.iml
+out
+gen
\ No newline at end of file
